### DISCUSSION TITLE
Litle typo in KAMP

### DISCUSSION THREAD

**Discussion by OxygenGS** [Guilouz/Creality-Helper-Script-Wiki#423] [General]
![Typo KAMP](https://github.com/Guilouz/Creality-K1-Series/assets/37304735/4fdfc90a-8012-473d-951f-4c80cdf26eb8)

It says here to type 1. Just a heads up :)

**Comment by Guilouz** [comment]
Yes Intall Menu is on choice 1, there is no error.

**Comment by OxygenGS** [comment]
Ow damn. I misread it. I was like: 1 is not to install KAMP XD
Sorry!
